Resources/config/mongodb.php

```diff
@@ -0,0 +1,91 @@
+        ->set('doctrine_mongodb.odm.cache.memcache_host', 'localhost')
+        ->set('doctrine_mongodb.odm.cache.memcache_port', 11211)
+        ->set('doctrine_mongodb.odm.cache.memcache_instance.class', 'Memcache')
+        ->set('doctrine_mongodb.odm.cache.xcache.class', 'Doctrine\Common\Cache\XcacheCache')
+        ->set('doctrine_mongodb.odm.connection.class', Client::class)
+        ->set('doctrine_mongodb.odm.configuration.class', Configuration::class)
+        ->set('doctrine_mongodb.odm.document_manager.class', DocumentManager::class)
+        ->set('doctrine_mongodb.odm.manager_configurator.class', ManagerConfigurator::class)
+        ->set('doctrine_mongodb.odm.class', ManagerRegistry::class)
+        ->set('doctrine_mongodb.odm.metadata.driver_chain.class', MappingDriverChain::class)
+        ->set('doctrine_mongodb.odm.metadata.attribute.class', AttributeDriver::class)
+        ->set('doctrine_mongodb.odm.metadata.xml.class', XmlDriver::class)
+        ->set('doctrine_mongodb.odm.mapping_dirs', [])
+        ->set('doctrine_mongodb.odm.xml_mapping_dirs', '%doctrine_mongodb.odm.mapping_dirs%')
+        ->set('doctrine_mongodb.odm.document_dirs', [])
+        ->set('doctrine_mongodb.odm.fixtures_dirs', []);
+
+    $containerConfigurator->services()
+
+        ->alias(DocumentManager::class, 'doctrine_mongodb.odm.document_manager')
+
+        ->alias(ManagerRegistry::class, 'doctrine_mongodb')
+
+        ->set('doctrine_mongodb.odm.connection.event_manager', ContainerAwareEventManager::class)
+            ->abstract(true)
+            ->args([
+                service('service_container'),
+            ])
+
+        ->set('doctrine_mongodb.odm.container_repository_factory', ContainerRepositoryFactory::class)
+            ->args([''])
+
+        ->set('doctrine_mongodb.odm.manager_configurator.abstract', ManagerConfigurator::class)
+            ->abstract(true)
+            ->args([
+                [],
+            ])
+
+        ->set('doctrine_mongodb.odm.security.user.provider', EntityUserProvider::class)
+            ->abstract(true)
+            ->args([
+                service('doctrine_mongodb'),
+            ])
+
+        ->set('doctrine_mongodb', '%doctrine_mongodb.odm.class%')
+            ->public()
+            ->args([
+                'MongoDB',
+                '%doctrine_mongodb.odm.connections%',
+                '%doctrine_mongodb.odm.document_managers%',
+                '%doctrine_mongodb.odm.default_connection%',
+                '%doctrine_mongodb.odm.default_document_manager%',
+                GhostObjectInterface::class,
+                service('service_container'),
+            ])
+
+        ->set('doctrine_mongodb.odm.listeners.resolve_target_document', ResolveTargetDocumentListener::class)
+
+        ->set('doctrine_mongodb.odm.symfony.fixtures.loader', SymfonyFixturesLoader::class)
+            ->args([
+                service('service_container'),
+            ]);
```

Comment:
The container injection was removed in [#802](https://github.com/doctrine/DoctrineMongoDBBundle/pull/802/files#diff-80a4f7479361b0ee5647b22762859618d9cd88b06d7fb9167fee7662d971507cL187)
```suggestion
        ->set('doctrine_mongodb.odm.symfony.fixtures.loader', SymfonyFixturesLoader::class);
```